administrator/components/com_patchtester/config.xml

```diff
@@ -21,29 +37,55 @@
 			label="COM_PATCHTESTER_FIELD_REPO_LABEL"
 			description="COM_PATCHTESTER_FIELD_REPO_DESC"
 			default="joomla-cms"
+			id="repo"
+			showon="repository:custom"
 		/>
 
+	</fieldset>
+
+	<fieldset
+		name="component"
```

Comment:
It's been a reoccurring issue for a while (I think there are core hacks in place to deal with Firefox having the same misbehavings).